src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements Stack{
+
+	private static final int CAPACITY = 12;
+
+	private final int[] numbers = new int[CAPACITY];
+
+	public int total = Stack.isEmptyIndicator;
```

Comment:
It is not Stack.isEmptyIndicator. It is another constant.  At the beginning total should be always -1 (like in the original code)